src/features/side_bar/components/SideBar/SideBar.tsx

```diff
@@ -143,7 +157,13 @@ export function SideBar({
                   <ActionsBarBTN
                     size={route.parentRoute ? 'small-xs' : 'small'}
                     active={currentRoute?.slug === route.slug}
-                    iconBefore={route.icon}
+                    iconBefore={
+                      route.parentRoute ? (
+                        <SmallIconSlot>{route.icon}</SmallIconSlot>
+                      ) : (
+                        route.icon
+                      )
+                    }
```

Comment:
It's good
Also it can be a component that handles the logic of what icon to return from route
```
<RouteIcon icon={route.icon} parent={route.parentRoute} ...etc/>
```
In case we need to handle cases like empty icon / capital letter icon or smth

Reply:
I think when we have more conditions and variations we will do so